### ISSUE TITLE
Points layer markers "expand" when switching from 2D to 3D view

### ISSUE THREAD

**Issue by VolkerH** [napari/napari#697]
## 🐛 Bug

Points added in 2D slice view appear much larger and artefacted when switching to 3d view

![image](https://user-images.githubusercontent.com/2210044/68928941-5269fa80-07df-11ea-9e01-d587b60dce1d.png)

## To Reproduce

1. open a volume
2. add a points layer
3. using 2D view with z slider add a few points and/or crosses at the default size
4. switch to 3D view

## Expected behavior

points retain relative size to voxel size when switching to 3D.
Note that for "Shapes" layers this is handled correctly (rectangles, triangles etc. maintain correct size in 3D view).

## Environment

napari, 0.2.5
Qt, 5.13.2
NumPy, 1.17.3
SciPy, 1.3.1
VisPy, 0.6.2
scikit-image, 0.16.2
Platform: Windows-10-10.0.18362-SP0
Python:   3.7.5 (default, Oct 31 2019, 15:18:51) [MSC v.1916 64 bit (AMD64)]
Backend:  PySide2
GL version:  '4.5.0 - Build 25.20.100.6472'
MAX_TEXTURE_SIZE: 16384

**Comment by VolkerH** [comment]
Steps to reproduce as video:

![points3D_napari](https://user-images.githubusercontent.com/2210044/68929600-c2c54b80-07e0-11ea-9ce1-eefc2d0877cd.gif)

**Comment by jni** [comment]
I can't reproduce on my machine, in the sense that it actually looks relatively good on yours compared to mine 😂 

<img width="1200" alt="Screen Shot 2019-11-15 at 8 14 08 pm" src="https://user-images.githubusercontent.com/492549/68931427-9f040480-07e4-11ea-8ad9-72e6497e16c3.png">

**Comment by jni** [comment]
I can reproduce with this script:

```python
import numpy as np
import napari
from skimage import data

blobs = data.binary_blobs(length=128, n_dim=3)

with napari.gui_qt():
    viewer = napari.view_image(blobs)
    points = viewer.add_points(np.array([[64, 64, 64], [0, 0, 127]]))
    viewer.dims.ndisplay=3
```

**Comment by VolkerH** [comment]
Just adding an observation. In my case the disks keep the same size on the screen when zooming in and out, but their centres change the position on the screen in what appears to be correct. Haven't looked into the code whether the size needs to be explicitly linked to the zoom or how these objects are handled.

**Comment by sofroniewn** [comment]
@jni @VolkerH - this is likely due to the release of vispy 0.6.2 where the archballcamera default changed to `fov=45` instead of `fov=0`. merging #683 should recover the previous behavior.

if we want to support proper scaling with the new fov then we need to look into some things on the vispy end - see #683 and some linked vispy discussions for more detail, but that will take longer

apologies for not getting this in sooner!

**Comment by sofroniewn** [comment]
not sure what's happening on @jni machine, if it is the same bug / unrelated / if he changed his vispy along the way

**Comment by sofroniewn** [comment]
This should now be fixed on master with the merge of #683 - can you confirm @VolkerH and then I will close this issue. Thanks for flagging!

**Comment by VolkerH** [comment]
I can confirm that the points now have a reasonable size in the 3D view.  Thanks, i will close the issue.
I noticed some other issue though (probably unrelated so I will open a new one).
